test/e2e/pages/DashboardPage.js

```diff
@@ -40,6 +40,15 @@ class DashboardPage {
 
   get modalText() { return $('.cu-dc__title');}
 
+  get floatingMenuButton() { return $('.cu-float-button__toggle'); }
+
+  get listViewHeader() { return $('cu-nav-menu>div>div>div>cu-editable'); }
+
+  // floatingButton(index) { return $(`div > .cu-float-button-item:nth-child(${index})`); }
```

Comment:
Is this line important to keep it in project?

Reply:
removed